README.md

```diff
@@ -199,3 +269,90 @@ format is CycloneDX so the `-f` isn't needed)
 ``` bash
 bomctl export <UUID> -o merged-sbom.cdx.json
 ```
+
+### SBOM Alias
+
+Working with Multiple SBOMs, it can be tricky to keep documents straight, as different formats have different expectations for naming. `bomctl`  
+typically relies on UUIDs for storing documents, however these can be taxing to use as an ID when performing operations. The `alias` command  
+gives the option to pick a more human readable name for a document for ease of use.
```

Comment:
Casing fix, semicolon instead of comma, and the UUID statement isn't quite accurate (yet) 

```suggestion
Working with multiple SBOMs, it can be tricky to keep documents straight, as different formats have different expectations for naming. `bomctl`  
typically relies on document IDs for storing documents; however, these can be taxing to use as an ID when performing operations. The `alias` command  
gives the option to pick a more human readable name for a document for ease of use.
```